src/App.css

```diff
@@ -87,3 +87,13 @@ BEM conventions we are using:
   margin: 0;
   list-style: none;
 }
+
+.visually-hidden {
+  clip: rect(0 0 0 0);
+  clip-path: inset(50%);
+  height: 1px;
+  overflow: hidden;
+  position: absolute;
+  white-space: nowrap;
+  width: 1px;
```

Comment:
I've never seen hiding an element defined with CSS in this way. Is there a benefit for doing this over something like `display: none`?

Reply:
Yep! It's used for situations where you want a screen reader to "see" something, but you don't want to display it visually (perhaps there is a visual design element already in place, but you want to provide an auditory fallback for people who can't see it). Screen readers can't see things styled as display: none, so it's necessarily to use this kind of weird hack. [Here's some more information from the a11y project.](https://www.a11yproject.com/posts/2013-01-11-how-to-hide-content/.)

Reply:
I had recently ready about this. So I thought it was kinda cool to see it in action.

Reply:
Thank you for linking the article @anderswift . Very helpful and informative!

Reply:
TIL 👀